src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1181,12 +1186,20 @@ public Location getCompassTarget()
 	{
 		return this.compassTarget;
 	}
+	/**
+	 * Sets the {@link InetSocketAddress} returned by {@link #getAddress}.
+	 *
+	 * @param address The address to set.
+	 */
+	public void setAddress(InetSocketAddress address)
+	{
+		this.address = address;
+	}
 
 	@Override
 	public InetSocketAddress getAddress()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return address;
```

Comment:
This method should return `null` if the `PlayerMock` is not currently "on" the server.